scopes/lanes/ui/hooks/lanes-query/lanes-query.ts

```diff
@@ -34,3 +53,28 @@ export function useLanesQuery(): { lanes?: LaneModel[] } & Omit<QueryResult<Lane
     lanes,
   };
 }
+
+export function useLaneComponentsQuery(
+  lane: LaneModel,
+  host: string
+): {
+  components?: Array<LaneComponentModel>;
+} & Omit<QueryResult<LanesQueryResult>, 'data'> {
+  const { data, ...rest } = useDataQuery(GET_LANE_COMPONENTS, {
+    variables: { name: lane.name },
+  });
+
+  const components: Array<LaneComponentModel> = data?.getLaneComponents.map((component) => {
+    const id = component && ComponentID.fromObject(component.id);
+
+    return {
+      model: ComponentModel.from({ ...component, host }),
```

Comment:
ok this is getting horrible. let's get the "host" data from the gql instead of the props (get it from getHost())